schema.sql

```diff
@@ -1,5 +1,10 @@
-/* Database schema to keep the structure of entire database. */
+CREATE DATABASE vet_clinic;
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT,
```

Comment:
## [OPTIONAL]
- [ ] To improve the animal table's id to not be null and be incrementing it self, I think it woul be better this code suggested below :point_down: 
#### suggested code 
```sql
id int NOT NULL IDENTITY(1,1)
```
#### your code 
```sql
 id INT,
```

Reply:
Thanks